[Login] When translating elements in loginForm, the first time, there is error in server console

### Problem Statement

There is an error in server log when saving translations first time in loginForm portlet

### Bug

Scenario :

    Create a new loginForm portlet instance
    Create a new page on dw portal for example
    Add the instance on the page. Save the page, and go to the page in view mode
    Click on the portlet edit button, and change at least 2 translations.
    Save preferences

Current Behaviour :

    There is an error in server logs

    Caused by: org.hsqldb.HsqlException: integrity constraint violation: unique constraint or index violation ; UC_SOC_METADATA_01 table: SOC_METADATAS at

Translations are correctly saved.

Expected :

    No errors in server logs.

### Feature — Solution & Expected Benefits (only if Type = Feature)

#### Proposed Solution
-

#### Expected Benefits
<!--
- Improves ...
- Reduces ...
- Measurable outcome: ...
-->

### Additional Context

MEED-9899

--- a/webapp/src/main/webapp/vue-apps/login-form/components/LoginFormSettingsDrawer.vue
+++ b/webapp/src/main/webapp/vue-apps/login-form/components/LoginFormSettingsDrawer.vue
@@ -262,7 +262,7 @@ export default {
     close() {
       this.$refs.drawer.close();
     },
-    save() {
+    async save() {
       this.loading = true;
       const promise = [];
       let firstTranslation = true;
@@ -273,7 +273,7 @@ export default {
 
       if (this.registerEnabled) {
         if (firstTranslation) {
-          this.saveTranslationSynchronously(this.objectType, this.translationIdentifier, 'newHere', this.translationsNewHere);
+          await this.$translationService.saveTranslations(this.objectType, this.translationIdentifier, 'newHere', this.translationsNewHere);
           firstTranslation = false;
         } else {
           promise.push(this.$translationService.saveTranslations(this.objectType, this.translationIdentifier, 'newHere', this.translationsNewHere));
@@ -281,7 +281,7 @@ export default {
         promise.push(this.$translationService.saveTranslations(this.objectType, this.translationIdentifier, 'createAccount', this.translationsCreateAccount));
       } else {
         if (firstTranslation) {
-          this.saveTranslationSynchronously(this.objectType, this.translationIdentifier, 'welcomeBack', this.translationsWelcomeBack);
+          await this.$translationService.saveTranslations(this.objectType, this.translationIdentifier, 'welcomeBack', this.translationsWelcomeBack);
           firstTranslation = false;
         } else {
           promise.push(this.$translationService.saveTranslations(this.objectType, this.translationIdentifier, 'welcomeBack', this.translationsWelcomeBack));
@@ -290,23 +290,25 @@ export default {
 
       if (this.signinOption === 'buttonform') {
         if (firstTranslation) {
-          this.saveTranslationSynchronously(this.objectType, this.translationIdentifier, 'signinEmailButton', this.translationsSigninEmailButton);
+          await this.$translationService.saveTranslations(this.objectType, this.translationIdentifier, 'signinEmailButton', this.translationsSigninEmailButton);
           firstTranslation = false;
         } else {
           promise.push(this.$translationService.saveTranslations(this.objectType, this.translationIdentifier, 'signinEmailButton', this.translationsSigninEmailButton));
         }
       }
 
-      this.providers.forEach((provider) => {
+      for (const provider of this.providers) {
         if (provider.translations && Object.keys(provider.translations).length > 0) {
           if (firstTranslation) {
-            this.saveTranslationSynchronously(this.objectType, this.translationIdentifier, provider.key, provider.translations);
+            /* eslint-disable no-await-in-loop */
+            await this.$translationService.saveTranslations(this.objectType, this.translationIdentifier, provider.key, provider.translations);
             firstTranslation = false;
+            /* eslint-enable no-await-in-loop */
           } else {
             promise.push(this.$translationService.saveTranslations(this.objectType, this.translationIdentifier, provider.key, provider.translations));
           }
         }
-      });
+      }
 
       promise.push(this.saveSettings());
       Promise.all(promise).then(() => {
@@ -324,9 +326,6 @@ export default {
         this.loading = false;
         this.close();
       });
-    },
-    async saveTranslationSynchronously(objectType, objectId, fieldName, translations) {
-      await this.$translationService.saveTranslations(objectType, objectId, fieldName, translations);
     },
     saveSettings() {
       const formData = new FormData();

